src/commands/txCommands/status.js

```diff
@@ -0,0 +1,21 @@
+const Rx = require('rxjs');
+const txlib = require('./util');
+const chalk = require('chalk');
+
+module.exports = {
+    command: 'status',
+    description: 'get translation status of resources (branches)',
+    handler: argv => {
+        txlib.checkEnvVars();
+        console.log(chalk.blue('checking resource status\n'));
+
+        console.log('Incomplete Resources')
```

Comment:
If there are no incomplete resources, will this just say

```js
checking resource status

Incomplete Resources
Completed Resources
...
```

?

Probably better to conditionally print `Incomplete Resources`